.cloudbees/workflows/workflow.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
     - name: docker build and publish
       uses: cloudbees-io/kaniko
       with:
-        destination: registry.saas-dev.beescloud.com/staging/helm-install-action:${{ cloudbees.scm.sha }}${{ cloudbees.scm.branch == 'main' && ',registry.saas-dev.beescloud.com/staging/helm-install-action:1.1.2,registry.saas-dev.beescloud.com/staging/helm-install-action:latest' || '' }}
+        destination: registry.saas-dev.beescloud.com/staging/helm-install-action:${{ cloudbees.scm.sha }}${{ cloudbees.scm.branch == 'main' && ',registry.saas-dev.beescloud.com/staging/helm-install-action:1.1.3,registry.saas-dev.beescloud.com/staging/helm-install-action:latest' || '' }}
```

Comment:
When you only change the action.yml, you don't need to bump the image version.

Reply:
although, this may be necessary now that we have to upgrade `skaffold` to test this fix properly.

Reply:
Rolled back this change.